test/jestLib/setup.js

```diff
@@ -1,7 +1,12 @@
-require('babel-polyfill')
+import Enzyme from 'enzyme'
+import Adapter from 'enzyme-adapter-react-16'
+
+import 'babel-polyfill'
 
 // polyfill for requestAnimationFrame
 /* istanbul ignore next */
 global.requestAnimationFrame = (cb) => {
   setTimeout(cb, 0)
 }
+
+Enzyme.configure({ adapter: new Adapter() })
```

Comment:
oh I didn't know configuring Enzyme inside jest setup would just work as simplely, thanks for this tip! ❤️